spec/injector_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'injector'
+
+describe Array do
+
+  it 'returns 8 when passed { |sum, n| } on [1, 5, 2]' do
+    expect([1, 5, 2].injector { |sum, n| sum + n }).to eq 8
+  end
+  
+  it 'returns "sheep" when passed ["cat", "sheep", "bear"]' do
+    ['cat', 'sheep', 'bear'].injector do |memo, word|
+	  memo.length > word.length ? memo : word 
+	end
```

Comment:
`end` at 12, 1 is not aligned with `['cat', 'sheep', 'bear'].injector do |memo, word|` at 10, 4<br>Tab detected.